doc/GettingStartedKubernetes.md

```diff
@@ -467,38 +552,44 @@ provides more detail about how the app server interacts with Vitess.
 
 ## Tear down and clean up
 
-The following command tears down the Container Engine cluster. It is
-necessary to stop the virtual machines running on the Cloud platform.
+Before stopping the Container Engine cluster, you should tear down the Vitess
+services. Kubernetes will then take care of cleaning up any entities it created
+for those services, like external load balancers.
+
+``` sh
+vitess/examples/kubernetes $ ./guestbook-down.sh
```

Comment:
Typically tearing down the cluster also tears down all the other services, is this not true in v1beta3 or are we just being more explicit here?